File: app/src/main/java/co/tpcreative/supersafe/common/request/VerifyCodeRequest.java
package co.tpcreative.supersafe.common.request;

import co.tpcreative.supersafe.common.util.Utils;

public class VerifyCodeRequest {
    public String code ;
    public String user_id;
    public String new_user_id;
    /*Supersafe _id*/
    public String _id;
    public String other_email;
    public String device_id;
    public String session_token;
    public VerifyCodeRequest(){
        this.session_token = Utils.getAccessToken();
    }
}

File: app/src/main/java/co/tpcreative/supersafe/common/request/VerifyCodeRequest.kt
package co.tpcreative.supersafe.common.request
import co.tpcreative.supersafe.common.util.Utils

class VerifyCodeRequest {
    var code: String? = null
    var user_id: String? = null
    var new_user_id: String? = null

    /*Supersafe _id*/
    var _id: String? = null
    var other_email: String? = null
    var device_id: String? = null
    var session_token: String?

    init {
        session_token = Utils.getAccessToken()
    }
}